db/schema.rb

```diff
@@ -12,7 +12,7 @@
 
 ActiveRecord::Schema.define(version: 2022_02_16_233321) do
 
-  create_table "account_facility_joins", id: :integer, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8", force: :cascade do |t|
+  create_table "account_facility_joins", id: :integer, charset: "utf8mb3", force: :cascade do |t|
```

Comment:
Why are these changing?

Reply:
@giladshanan this was an automatic change from `rails app:update`.

It looks like "utf8" is a lie in MySQL and is an alias for "utf8mb3", which is 3 bytes instead of 4, so I believe this is just making that explicit. True utf8 is actually called "utf8mb4."

https://dev.mysql.com/doc/refman/8.0/en/charset-unicode-sets.html

https://adamhooper.medium.com/in-mysql-never-use-utf8-use-utf8mb4-11761243e434

Reply:
Interesting.  Did you run `rails db:migrate` to see if these changes persist?